api/tests/organizational-entities/unit/infrastructure/serializers/jsonapi/certification-center/certification-center-for-admin.serializer.test.js

```diff
@@ -45,6 +46,8 @@ describe('Unit | Organizational Entities | Infrastructure | Serializer | JSONAPI
 
   describe('#deserialize', function () {
     it('should convert a JSON API data into a CertificationCenterForAdmin model object', function () {
+      // given
+      certificationCenterJsonApi.createdBy = 123;
       // when
       const deserializedCertificationCenterForAdmin = serializer.deserialize(certificationCenterJsonApi);
```

Comment:
Il faut mettre à jour ce test avec la nouvelle signature de la fonction `deserialize` comme suit, tout en notant que cela casse ensuite des tests et qu'il va falloir faire évoluer un peu le fichier : 
```suggestion
      // given
      const createdBy = 123;

      // when
      const deserializedCertificationCenterForAdmin = serializer.deserialize({ data: certificationCenterJsonApi, createdBy });
```